proxy/config_test.go

```diff
@@ -94,6 +94,32 @@ func TestWhenHasErgoFile(t *testing.T) {
 
 	})
 
+	t.Run("It removes a service", func(tt *testing.T) {
+		fileContent, err := ioutil.ReadFile("../ergo")
+
+		if err != nil {
+			tt.Skipf("Could not load initial .ergo file")
+		}
+
+		//we clean after the test. Otherwise the next test will fail
+		defer ioutil.WriteFile("../.ergo", fileContent, 0755)
+
+		service := NewService("testservice", "http://localhost:8080")
+
+		if err := RemoveService("../.ergo", service); err != nil {
+			tt.Errorf("Expected service to be removed")
```

Comment:
Maybe a better explanation here would be something like. "_Expected no error while removing service. Got %s_". This way we also have different messages from the one at line 119.

Reply:
Used "Expected no error while removing service. Got %v\n"